lib/jekyll/converters/scss.rb

```diff
@@ -214,15 +200,27 @@ def sass_convert(content)
         raise SyntaxError, e.to_s
       end
 
+      def sass_embedded_config
+        {
+          :load_paths                 => sass_load_paths,
+          :source_map                 => sourcemap_required?,
+          :source_map_include_sources => true,
+          :style                      => sass_style,
+          :syntax                     => syntax == :sass ? :indented : syntax,
+          :url                        => url,
```

Comment:
Would like this to be `local_url` or `file_url` or similar for greater comprehensibility.
```suggestion
          :url                        => sass_file_url,
```